requirements.txt

```diff
@@ -14,3 +14,4 @@ mdf-connect-client>=0.4.0
 json2table>=1.1.5
 joblib>=1.1.0
 torch>=1.8.0
+tensorflow>=2.8
```

Comment:
I strongly vote against adding ML packages as mandatory requirements, as they can be large and difficult to get combined together with other packages. 

We can do optional, or just leave them out and inform users they must install tesnorflow when they want to use specific features.